furusiya/main.py

```diff
@@ -6,16 +6,65 @@
 
 # Has to be here, because we use it everywhere
 from model.config import file_watcher, config
+
 file_watcher.watch('config.json', lambda raw_json: config.load(raw_json))
 
-from function_mess import new_game, play_game, load_game
-from main_interface import Game
-from constants import SCREEN_WIDTH, SCREEN_HEIGHT
+from view.map_renderer import MapRenderer
+from model.entities.party.player import Player
+from model.entities.party.stallion import Stallion
+from model.maps import generators
+from model.maps.area_map import AreaMap
+from model.maps.generators import DungeonGenerator
+from main_interface import Game, message
+from constants import SCREEN_WIDTH, SCREEN_HEIGHT, MAP_WIDTH, MAP_HEIGHT
 import random
 
 
+def new_game():

```

Comment:
Needs tests: at least that:
-  `Game.area_map` is initialized with a non-zero size
- `Game.player` is not `None`
- ditto for stallion (you can also check the coordinates are close to the player)
- player coordinates are random and walkable (call `new_game` twice and he should move)
- Inventory is not `None`, it's an empty list
- game state is `playing`
- Game messages are empty